app/models/repository.rb

```diff
@@ -74,6 +74,21 @@ def self.handle_push_event(event)
     repository
   end
 
+  # Handle a delete event.
+  def self.handle_delete_event(event)
+    registry = Registry.find_from_event(event)
+    return if registry.nil?
+
+    # Fetch the repo.
+    ns, repo_name, = registry.get_namespace_from_event(event, false)
```

Comment:
Can you write `ns, repo_name, _ = registry.get_namespace_from_event(event, false)` please? Than it's more obvious that this method returns 3 values but you don't care about the last one.